third_party/gpus/cuda_configure.bzl

```diff
@@ -403,12 +403,16 @@ def _create_cuda_repository(repository_ctx):
        })
 
   # Set up crosstool/
+  cuda_include_path = '%s-%s/include' % (cuda_toolkit_path, cuda_version)
```

Comment:
Can you add a comment describing the non-standard CUDA toolkit path that makes this necessary?

Also, it would be better to move this into a helper function, similar to `_find_cudnn_header_dir()`.